packetbeat/protos/pgsql/parse.go

```diff
@@ -265,6 +271,53 @@ func (pgsql *Pgsql) parseErrorResponse(s *PgsqlStream, length int) (bool, bool)
 	return true, true
 }
 
+func (pgsql *Pgsql) parseParse(s *PgsqlStream, length int) (bool, bool) {
+	// Ready for query -> Parse for an extended query request
+	detailedf("Parse")
+
+	m := s.message
+	m.start = s.parseOffset
+	m.IsRequest = true
+
+	s.parseOffset += 1 //type
+	s.parseOffset += length
+	m.end = s.parseOffset
+	m.Size = uint64(m.end - m.start)
+	m.toExport = true
+
+	query, err := common.ReadString(s.data[m.start+6:])
+	if err != nil {
+		detailedf("Invalid extended query request")
+		return false, false
+	}
+	m.Query = query
+	detailedf("Parse in an extended query request: %s", m.Query)
+
+	// Ignore SET statement
+	if strings.HasPrefix(m.Query, "SET ") {
+		m.toExport = false
+	}
+	s.parseState = PgsqlExtendedQueryState
+	return pgsql.parseMessageExtendedQuery(s)
+}
+
+func (pgsql *Pgsql) parseParseCompletion(s *PgsqlStream, length int) (bool, bool) {
```

Comment:
maybe call it `parseExtCompl` or 'parseExtResp` to make clear this method is about handling extended queries?